docs/src/pages/index.astro

```diff
@@ -1,251 +1,9 @@
-}
-
-.npm-init-snippet {    
-    font-family: 'Roboto Mono', monospace;
-    font-size: 1.6rem;
-    background-color: var(--theme-code-inline-bg);
-    color: var(--theme-text);
-    border-radius: 6px;
-    padding: 4px 14px;
-    margin-top: 1rem;
-}
-
-      .content {
-        width: 100%;
-        height: 100%;
-        display: flex;
-        flex-direction: column;
-        align-items: center;
-        justify-content: center;
-      }
-
-      .content > main {
-        margin-bottom: 4rem;
-      }
-
-      #sidebar-content {
-        display: none;
-      }
-      .theme-toggle-wrapper {
-        display: none;
-      }
-
-      #sidebar-site {
-        display: none;
-      }
-      :global(.mobile-sidebar-toggle) {
-        overflow: hidden;
-      }
-      :global(.mobile-sidebar-toggle) #sidebar-site {
-        display: block;
-      }
-      @media (min-width: 60em) {
-        #sidebar-site {
-          display: flex;
-        }
-        :global(.mobile-sidebar-toggle) {
-          overflow: initial;
-        }
-        :global(.mobile-sidebar-toggle) #sidebar-site {
-          display: flex;
-        }
-        .menu-toggle {
-          display: none;
-        }
-        .layout {
-          grid-template-columns: 
-            20rem 
-            minmax(0, 1fr);
-        }
-        #article {
-          grid-column: 2;
-        }
-        #sidebar-site {
-          position: sticky;
-        }
-        #sidebar-nav {
-          display: flex;
-        }
-        .theme-toggle-wrapper {
-          display: flex;
-        }
-      }
-
-      @media (min-width: 82em) {
-        .layout {
-          grid-template-columns: 
-            20rem 
-            minmax(0, var(--max-width)) 
-            18rem;
-          padding-left: 0;
-          padding-right: 0;
-          margin: 0 auto;
-        }
-
-        #sidebar-nav {
-          grid-column: 1;
-        }
-        #article {
-          grid-column: 2/4;
-        }
-      }
-
-
-    </style>
-  </head>
-
-  <body class="mobile-sidebar-hidden">
-    <header>
-      <nav class="nav-wrapper">
-        <div class="menu-and-logo flex">
-          <div class="menu-toggle">
-              <MenuToggle client:idle/>
-          </div>
-          <a id="site-title" href="/">
-            <h1>Astro Documentation</h1>
-          </a>
-        </div>
-
-        <div />
-
-        <div class="theme-toggle-wrapper">
-          <ThemeToggle client:idle />
-        </div>
-      </nav>
-    </header>
-
-    <main class="layout">
-      <aside class="sidebar" id="sidebar-site">
-        <SiteSidebar currentPage="" />
-      </aside>
-      <div id="article">
-        <article class="content">
-          <AstroLogo size={160} />
-          <div class="npm-init-snippet">
-            npm init astro
-          </div>
-        </article>
-      </div>
-    </main>
-  </body>
-</html>
+<Layout centered>
+  <h1>
+    <code>npm init astro</code>
```

Comment:
This `<code>` needs to be `display: block` or wrapped in a `<pre>` or something to prevent it from wrapping.
<img width="488" alt="image" src="https://user-images.githubusercontent.com/10626596/126243986-5895ca20-87e4-4bee-83aa-f2c06f1e6f23.png">